helmfile/charts/ingress-nginx-probe-ingress/values.yaml

```
@@ -0,0 +1,8 @@
+nameOverride: ""
+
+service:
+  name: "ingress-nginx-controller-metrics"
+  port: 10254
+
+ingress:
```

Might want to have some default for the `host` and `annotation` variable here as well

```suggestion
ingress:
  host: ingress.example.com
  annotations: {}
```